store-and-retrieve-data/use-storagehub-sdk/get-started.md

```diff
@@ -78,6 +78,139 @@ Add the core and MSP client packages to your project. These libraries provide th
     npm install @storagehub-sdk/core @storagehub-sdk/msp-client
     ```
 
+## Initialize the StorageHub SDK
+
+In this step, you will learn how to initialize all the clients you will need while building with the StorageHub SDK. You will go through installing extra dependencies, setting up `walletClient`, `publicClient`, `storageHubClient`, `polkadotApi`, `mspClient`, and a few helper functions related to the `mspClient` to help get you started with things like checking MSP health and user authentication.
+
+### Install Additional Dependencies
+
+=== "pnpm"
+
+    ```bash { .break-spaces }
+    pnpm add @storagehub/types-bundle @polkadot/api @storagehub/api-augment viem
+    ```
+
+=== "yarn"
+
+    ```bash { .break-spaces }
+    yarn add @storagehub/types-bundle @polkadot/api @storagehub/api-augment viem
+    ```
+
+=== "npm"
+
+    ```bash { .break-spaces }
+    npm install @storagehub/types-bundle @polkadot/api @storagehub/api-augment viem
+    ```
+
+??? interface "Why do I need these dependencies?"
+
+    - **[`@storagehub/types-bundle`](https://www.npmjs.com/package/@storagehub/types-bundle){target=_blank}:** Describes DataHaven's custom on-chain types.
+
+    - **[`@polkadot/api`](https://www.npmjs.com/package/@polkadot/api){target=_blank}:** The core JavaScript library used to talk to any Substrate-based blockchain, which in our case is DataHaven.
+
+    - **[`@storagehub/api-augment`](https://www.npmjs.com/package/@storagehub/api-augment){target=_blank}:** Extends `@polkadot/api` with DataHaven's custom pallets and RPC methods. You will import it in your `index.ts` file where your main script logic will be executed.
+
+    - **[`viem`](https://www.npmjs.com/package/viem){target=_blank}:** Lightweight library for building Ethereum-compatible applications.
+
+### Set Up Client Service
```

Comment:
How we're breaking these steps up doesn't make too much sense to me because we add the viem clients into step 2 without any focus on it and then there's a specific step for storagehub and polkadot API.

I think we could actually just drop the entire `clientServices.ts` file into step 2 with all the clients initialized in one go. Then we could remove "view the complete file" and avoid the mess of copying and pasting at different stages.

But also if we do want to break it out, which I think could potentially be helpful cause maybe not everyone wants to use viem, I would say create a separate step for adding in the viem clients.

Pros and cons 🤷‍♀️  I'll defer to you here 🙂

Reply:
My logic was to focus on "our new stuff" and not on "viem stuff" which is already an established library (yes, that's assuming the dev reading the guide is familiar with viem), but I do agree it should be one way or the other, not halfway. 

I think the entire `clientService.ts` file in bulk makes more sense since it simplifies the page, removes the expandable, makes copying easier, and all the explanations will still be there via comments. If someone prefers a non-viem approach I don't think they'll have trouble editing that part of the code.